{{ cookiecutter.name }}/.github/workflows/ci.yml

```
@@ -0,0 +1,94 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: build
+        uses: ./.github/actions/build
+
+      - name: lint
+        run: make lint
+
+  test:
+    needs: lint
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:16.1-alpine
+        env:
+          POSTGRES_PASSWORD: secret
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+      redis:
+        image: redis:7.2.3-alpine
+        ports:
+          - 6379:6379
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: build
+        uses: ./.github/actions/build
+
+      - name: install locale stuff
+        uses: awalsh128/cache-apt-pkgs-action@v1
+        with:
+          packages: locales-all gettext
+          version: 1
+
+      - name: get number of cpu cores
+        uses: SimenB/github-actions-cpu-cores@v2.0.0
+        id: cpu-cores
+
+      - name: test
+        env:
+          DATABASE_URL: postgres://postgres:secret@localhost:5432/postgres
+        run: make test -e SIMULTANEOS_TEST_JOBS=${{ steps.cpu-cores.outputs.count }}
+
+  build-docker-image:
+    needs: test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: build
+        uses: ./.github/actions/build
+
+      - name: setup qemu
+        uses: docker/setup-qemu-action@v2
+
+      - name: setup buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: make sure docker image is buildable
+        uses: docker/build-push-action@v3
```

```suggestion
        uses: docker/build-push-action@v5
```

Правда я эту версию ещё нигде не тестил